searx/engines/qwant.py

```diff
@@ -144,8 +146,8 @@ def response(resp):
         mainline_items = row.get('items', [])
         for item in mainline_items:
 
-            title = item['title']
-            res_url = item['url']
+            title = item.get('title', None)
+            res_url = item.get('url', None)
```

Comment:
minor comment: it can be `res_url = item.get('url')` (without `None` since this is the default value)
See: https://docs.python.org/3/library/stdtypes.html#dict.get

Reply:
> without None since this is the default value

Thanks, I will take this in mind for the future.